core/lexicon/en/tv_widget.inc.php

```diff
@@ -110,7 +110,7 @@
 $_lang['resourcelist_parents'] = 'Parents';
 $_lang['resourcelist_parents_desc'] = 'A list of IDs to grab children for the list.';
 $_lang['resourcelist_where'] = 'Where Conditions';
-$_lang['resourcelist_where_desc'] = 'A JSON object of where conditions to filter by in the query that grabs the list of Resources. (Does not support TV searching.)';
+$_lang['resourcelist_where_desc'] = 'A JSON object of where conditions to filter by in the query that grabs the list of Resources. (Does not support TV searching.)<br/>Examples: [{"template:=":"4"}], [{"pagetitle:!=":"Home"}], [{"parent:IN":[34,56]}]';
```

Comment:
I just took the examples from http://www.sepiariver.ca/blog/modx-web/modx-quick-tip-resource-list-tvs/. Did not check if there is a difference in behaviour if you write `{"template:=":4}` instead of `{"template:=":"4"}`. Have to check that - but currently I think there is no difference due to PHPs type juggling. But you are right, examples here should be as specific as possible.
Btw: any other, maybe more complex examples are welcome.